Revoke workspace deletion request feature (#7372)
segmentio/segment-docs: src/guides/usage-and-billing/account-management.md

## How do I delete my workspace entirely?

> To delete your workspace, go to your [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"}, click the **General Settings** tab, then click **Delete Workspace**. After a 5 day cool-off period, Segment will irrevocably delete your workspace. 

Author: ```suggestion
To delete your workspace, go to your [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"}, click the **General Settings** tab, then click **Delete Workspace**.  Segment will irrevocably delete your workspace 5 days after you initiate your deletion request. 
```

> If you want to revoke the workspace deletion request during the 5 day cool-off period, open the [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"} page, select the **General Settings** tab and click **Revoke Workspace Deletion**. 

Author: ```suggestion
If you want to revoke the workspace deletion request during the 5 days after you initiated your request, open the [Workspace Settings](https://app.segment.com/goto-my-workspace/settings/basic){:target="_blank"} page, select the **General Settings** tab and click **Revoke Workspace Deletion**. 
```
